src/app/workflow/version-modal/version-modal.service.ts

```diff
@@ -74,6 +78,8 @@ export class VersionModalService {
     if (workflowMode !== 'HOSTED') {
       this.workflowsService.updateWorkflowVersion(workflowId, [workflowVersion]).subscribe(
         (response) => {
+          workflow = { ...workflow, workflowVersions: response };
```

Comment:
It looks  like we don't have this [lint rule](https://eslint.org/docs/rules/no-param-reassign) turned on, but I think it's a good practice and would recommend not reassigning the parameter value. But your code will work as is.